docs/source/api_reference/image_video_overlay.rst

```diff
@@ -44,6 +44,6 @@ Attributes
 ===========    ========================   ===
 Attribute      Default Value              Doc
 ===========    ========================   ===
-url            ""                         An http url to the footage or a relative path to a local file (image/video). Note that absolute local paths are not supported. 
+url            ""                         An http url to the footage or a relative path to a local file (image/video). Note that absolute local paths are not supported  (e.g. ‘image.png’ or ‘files/working_dir/image.png’ in JupyterLab). 
```

Comment:
I don't quite understand, this makes it not clear if `image.png` and `files/working_dir/image.png` are supported or not, and in JupyterLab or not.

Reply:
There are 2 spaces in: "supported  ("

Reply:
Ok thanks for the comment I just made another PR to improve this let me know if it is ok.
New text:
> An http url to the footage or a relative path to a local file (image/video, e.g. ‘image.png’ or ‘files/working_dir/image.png’ in JupyterLab). Note that absolute local paths are not supported.

Reply:
Maybe:

> An http url to the footage or a relative path to a local file (e.g. ‘image.png’ or ‘files/working_dir/image.png’ in JupyterLab). Note that absolute local paths are not supported.

Reply:
Ok let's close this one then.

Reply:
You didn't have to open a new PR, you could push new commits to this one.

Reply:
Ok sorry, just noticed it.
Thanks